_sass/layouts/_posts.scss

```diff
@@ -17,6 +17,23 @@ article {
     font-size: 0.9em;
   }
 }
+footer {
+  width: 100%;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-size: 0.9em;
+  display: inline-block;
+  text-align: center;
+  .meta {
```

Comment:
The `.meta` rule shouldn't be copied and pasted. Can we refactor the `.meta` rule used in the `header` instead.